### PR TITLE
NIO Task 1

### PR DESCRIPTION
**Pull request by Tiger656**

### CODE REVIEW

File: src/test/java/com/epam/mjc/io/FileReaderTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
+package com.epam.mjc.io;
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/io/FileReaderTest.java]: As I can see, this task for NIO, but package called `io`

File: src/test/java/com/epam/mjc/io/FileReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    private final static Path READER_CLASS = Path.of("src/main/java/com/epam/mjc/io/FileReader.java");
+    private static final String PROFILE_PARAM_VALUE_DELIMETER = ": ";
+    private static final String SOURCE_CODE_LOCATION = "src/main/java";
+
+
+    @Test
+    public void testProgramReadsDataCorrectly () {
+        FileReader fileReader = new FileReader();
+        Profile actual = fileReader.getDataFromFile(new File(TEST_FILE.toUri()));
+        Profile expected = readFile(TEST_FILE);
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/io/FileReaderTest.java]: Sorry, but student can look into this file and copy logic from here.

File: src/test/java/com/epam/mjc/io/FileReaderTest.java (hunk 1 above)
Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: I agree

File: src/test/java/com/epam/mjc/io/FileReaderTest.java (hunk 2 above)
Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: Any ideas?

Comment (egoist301) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: You can create Profile:
`Profile expected = new Profile(name, phone...);`
And I suggest duplicate `.txt` file and move one in `test/resources` or create another file in this directory.

Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: Simple, but effective. I agree

Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: Fixed

File: src/test/java/com/epam/mjc/io/FileReaderTest.java (hunk 1 above)
Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: Fixed

**Review by egoist301 — APPROVED** [review]

**Pull request merged**